navigation.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  createLocalizedPathnamesNavigation,
+  Pathnames
+} from 'next-intl/navigation';
+ 
+export const locales = ['en', 'ar'] as const;
+export const localePrefix = 'always'; // Default
+ 
+// The `pathnames` object holds pairs of internal
+// and external paths, separated by locale.
+export const pathnames = {
+  // If all locales use the same pathname, a
+  // single external path can be provided.
+  '/': '/',
+  '/login': '/login',
+  '/register': '/register',
+  '/yousef':'/yousef',
+  'forgot-password':'forgot-password',
+  'reset-password':'reset-password',
+  
+  // If locales use different paths, you can
+  // specify each external path per locale.
+  '/about': {
+    en: '/about',
+    ar: '/ueber-uns'
+  },
```

Comment:
Why `/ueber-uns`

Reply:
I didn't use this path, but I let it in the file to use something like it when I need

Reply:
> when I need

Will you need it?

Reply:
I don't know if I will need something like it, or not